.travis.yml

```diff
@@ -14,11 +18,11 @@ jobs:
   include:
   - stage: test
     script:
-    - cd $HOME/gopath/src/github.com/integr8ly/keycloak-operator && make setup test
+    - cd $HOME/gopath/src/github.com/integr8ly/$OPERATOR_NAME && make setup test
   - stage: deploy
     script:
     - docker login --password "$QUAY_PASSWORD" --username "$QUAY_USERNAME" quay.io
-    - cd $HOME/gopath/src/github.com/integr8ly/keycloak-operator
-    - make compile COMPILE_TARGET=./tmp/_output/bin/keycloak-operator
-    - docker build -t quay.io/integreatly/keycloak-operator:$(git rev-parse --short ${TRAVIS_COMMIT}) -f tmp/build/Dockerfile .
-    - docker push quay.io/integreatly/keycloak-operator:$(git rev-parse --short ${TRAVIS_COMMIT})
+    - cd $HOME/gopath/src/github.com/integr8ly/$OPERATOR_NAME
+    - make compile COMPILE_TARGET=./tmp/_output/bin/$OPERATOR_NAME
+    - docker build -t quay.io/integreatly/$OPERATOR_NAME:$(git rev-parse --short ${TRAVIS_COMMIT}) -f tmp/build/Dockerfile .
```

Comment:
Should you not be using the operator-sdk build here https://github.com/integr8ly/keycloak-operator/blob/master/Makefile#L39

Reply:
I chose not to do that, as it is not installed in the travis environment by default, and digging into the operator, the above command is all that `operator-sdk build` does anyway.

Reply:
the operator-sdk is not installed, I mean.

Reply:
Yep OK, i see